src/v2/guide/components.md

```diff
@@ -334,7 +334,7 @@ When we click on the button, we need to communicate to the parent that it should
 ></blog-post>
 ```
 
-Then the child component can emit an event on itself by calling the built-in [**`$emit`** method](../api/#vm-emit), passing the name of the event:
+Then the child component can emit an event on itself by calling the built-in [**`$emit`** method](../api/#vm-emit-eventName-…args), passing the name of the event:
```

Comment:
oddly, in the netlify preview, and now on live, the anchor is `#vm-emit`: https://deploy-preview-2902--vuejs.netlify.app/v2/api/#vm-emit

https://vuejs.org/v2/api/#vm-emit

this must have been fixed by some other change?